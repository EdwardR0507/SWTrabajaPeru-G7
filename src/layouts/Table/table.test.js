/*Importamos las librerias principales*/
import React from 'react';
import { shallow } from 'enzyme';
import Table from "@material-ui/core/Table";
import CustomizedTables from '../../layouts/Table/index';
/*Renderizado de la vista SignUp*/
describe("<CustomizedTables></CustomizedTables>", () => {
  it('render "CustomizedTables"', () => {
    const wrapper = shallow(<CustomizedTables />);
    expect(wrapper).toMatchSnapshot()
  });
  it('should have the `Table` in "CustomizedTables"', () => {
    const wrapper = shallow(<CustomizedTables />);
    expect(wrapper.containsMatchingElement(<Table></Table>)).toBe(true);
  }); 
});