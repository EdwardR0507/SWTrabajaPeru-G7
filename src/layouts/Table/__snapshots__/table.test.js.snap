// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomizedTables></CustomizedTables> render "CustomizedTables" 1`] = `ShallowWrapper {}`;
